example/src/PageRectangleRoiMobileDemo.js

```diff
@@ -34,19 +35,38 @@ const rectangleRoiMobileToolState = {
   ],
 };
 
-const PageRectangleRoiMobileDemo = function () {
-  useDebugToolState()
+const PageRectangleRoiMobileDemo = function() {
+  useDebugToolState();
 
+  const [activeToolName, setActiveToolName] = useState(ReactCornerstoneViewportHooksHelpers.TOOL_NAMES.RectangleRoiMobile);
+
+  const onClick = toolName => () => setActiveToolName(toolName);
   return (
     <div className="container">
       <h5>
         <Link to="/">Back to Examples</Link>
       </h5>
 
       <h2>Rectangle Roi Mobile Demo</h2>
+      <button
+        style={{
+          backgroundColor:
+            activeToolName === "Zoom"
+              ? "tomato" : "",
+        }}
+        onClick={onClick("Zoom")}>Zoom
+      </button>
+      <button
+        style={{
+          backgroundColor:
+            activeToolName === ReactCornerstoneViewportHooksHelpers.TOOL_NAMES.RectangleRoiMobile
```

Comment:
What's with the `tomato`?

Reply:
Tomato is valid webcolor.
It show which tool is active